resources/views/applicant/application_index/application.html.twig

```diff
@@ -16,7 +16,7 @@
             </div>
             <div data-c-grid-item="tl(1of3)" data-c-align="base(left) tl(right)">
                 {# Status #}
-                {% if item.application_status.name != "draft"  %}
+                {% if not item.isDraft()  %}
```

Comment:
This kind of thing we need to be careful with (both the old and new syntax) as it's running the query on the application_status table for every loop iteration. We should add an eager load in the Controller or a `with` array in the model itself

Reply:
Good call. I'll set up an issue for this.